spec/models/user_spec.rb

```diff
@@ -644,48 +577,15 @@
     end
   end
 
-  describe '#password_updated_at' do
-    let(:user) { users(:affiliate_admin) }
-
-    it 'is set when the user is created' do
-      expect(user.password_updated_at).to_not be_nil
-    end
-
-    it 'is set when the password is updated' do
-      expect { user.update(password: "new1234!") }.
-        to change{ user.password_updated_at }
-    end
+  describe '#self.from_omniauth' do
+    subject(:from_omniauth) { OmniAuth.config.mock_auth[:login_dot_gov] }
```

Comment:
This is not actually testing the `User.from_omniauth` method. To do that, you'd need to change this to something like:
```ruby
let(:auth) { OmniAuth.config.mock_auth[:login_dot_gov] }
subject(:from_omniauth) { User.from_omniauth(auth) }
```